Explorer/Assets/DCL/PluginSystem/World/AssetsCollidersPlugin.cs

```diff
@@ -17,6 +19,11 @@ public class AssetsCollidersPlugin : IDCLWorldPluginWithoutSettings
         private readonly IPhysicsTickProvider physicsTickProvider;
         private readonly IComponentPoolsRegistry componentPoolsRegistry;
 
+        static AssetsCollidersPlugin()
+        {
+            EntityEventBuffer<PrimitiveColliderComponent>.Register(500);
```

Comment:
I would extract this `500` value into a constant in order to have it more visible in case we want to change this param in the future.

Reply:
good idea. I just copy-pasted it from other examples